user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,103 @@
+$ sudo shutdown -r 0
+```
+
+This will immediately restart the machine. `travis-worker`, the program which actually runs the builds, is configured to start automatically on system startup.
+
+#### Configuration
+
+Please check if the worker machine has all relevant configuration in order. To do so, please use ssh to login to the machine, then open `/etc/default/travis-enterprise`. This is the main configuration file `travis-worker` uses to connect to the platform machine. Below you find an example:
+
+```
+# Default ENV variables for Travis Enterprise
+# Uncomment and set, then restart `travis-worker` for
+# them to take effect.
+
+export TRAVIS_ENTERPRISE_BUILD_ENDPOINT="__build__"
+export TRAVIS_ENTERPRISE_HOST="travisci.example.com"
+export TRAVIS_ENTERPRISE_SECURITY_TOKEN="abc12345"
+# export TRAVIS_WORKER_DOCKER_PRIVILEGED="true"
+```
+
+Relevant are `TRAVIS_ENTERPRISE_HOST` and `TRAVIS_ENTERPRISE_SECURITY_TOKEN`. The former needs to contain your primary domain you use to access the Travis CI Enterprise Web UI. `travis-worker` uses this domain to reach the platform machine. The value of the latter needs to match the `RabbitMQ Password` on `https://yourdomain.com:8800/settings`. If you have made changes to this file, please run the following so they take effect:
+
+```bash
+$ sudo restart travis-worker
+```
+
+#### Ports are not open Security groups / firewall
+
+A source for the problem could be that the worker machine is not able to talk to the platform machine.
+Here we're distinguishing between an AWS EC2 installation and an installation running on other hardware. For the former, security groups need to be configured per machine. To do so, please follow our installation instructions [here](https://docs.travis-ci.com/user/enterprise/installation/#1.1.-Create-a-Security-Group). If you're not using AWS EC2, please make sure that the ports listed [in the docs](https://docs.travis-ci.com/user/enterprise/installation/#1.1.-Create-a-Security-Group) are open in your firewall.
+
+If none of the steps above lead to results for you, please follow the steps in [#Contact-support](#Contact-support) to move forward.
+
+## Contact support
+
+To get in touch with us, please write a message to [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com). In your message we'd like to ask you to answer the questions below.
+
+- What is the problem?
+- Which steps did you try already?
+- A support bundle (You can get it from https://yourdomain:8800/support)
+- Worker log files (They can be found at `/var/log/upstart/travis-worker.log`) - If you're using multiple worker machines, we need the log files from all of them.
+
+Is anything special with your setup? There are certain information we can already see, such as the hostname, which IaaS provider you're using, but there are lots of other things we can't see which could lead to something not working. Therefore we'd like to ask you to also answer the questions below in your support request (if applicable):
```

Comment:
One note:
> There are certain information we can already see, such as the hostname

I'd qualify this a little, because it's not _always_ possible: 
> While we may be able to see some information (such as hostname, or IaaS provider)...